driver/pelican/pelican.go

```diff
@@ -169,6 +190,50 @@ func (pel *Pelican) GetStatus() (*PelicanStatus, error) {
 		}
 	}
 
+	// Thermostat History Object Request to retrieve time stamps from past hour
+	timezone, timeErr := time.LoadLocation(pel.timezone)
+	if timeErr != nil {
+		return nil, fmt.Errorf("Invalid Timezone specified in pelican struct: %v\n", timeErr)
+	}
+	diff := time.Now().Add(-1 * time.Hour)
+	endTime := time.Now().In(timezone).Format(time.RFC3339)
+	startTime := diff.In(timezone).Format(time.RFC3339)
+
+	respHist, _, errsHist := pel.req.Get(pel.target).
+		Param("username", pel.username).
+		Param("password", pel.password).
+		Param("request", "get").
+		Param("object", "ThermostatHistory").
+		Param("selection", fmt.Sprintf("startDateTime:%s;endDateTime:%s;", startTime, endTime)).
+		Param("value", "timestamp").
+		End()
+	defer respHist.Body.Close()
+
+	if errsHist != nil {
+		return nil, fmt.Errorf("Error retrieving thermostat status from %s: %v", pel.target, errsHist)
+	}
+
+	var histResult apiResultHistory
+	histDec := xml.NewDecoder(respHist.Body)
+	if histErr := histDec.Decode(&histResult); histErr != nil {
+		return nil, fmt.Errorf("Failed to decode response XML: %v", histErr)
+	}
+	if histResult.Success == 0 {
+		return nil, fmt.Errorf("Error retrieving thermostat status from %s: %s", respHist.Request.URL, histResult.Message)
+	}
+
+	if len(histResult.Records.History) == 0 {
+		return nil, nil
+	}
+
+	// Converting string timeStamp to int64 format
+	match := histResult.Records.History[len(histResult.Records.History)-1]
+	timeString := match.TimeStamp + ":00Z"
```

Comment:
Ah I see, RFC 3339 is a stricter subset of ISO 8601 that requires the timezone to be specified.

Okay, I think you'll need to pass a custom format string to `time.Parse` instead of relying on the built-in `time.RFC3339`.